.circleci/config.yml

```diff
@@ -121,6 +121,27 @@ jobs:
           name: Reporting code coverage...
           command: bash <(curl -s https://codecov.io/bash) -F << parameters.package >> -X gcov
 
+  lint:
+    resource_class: medium+
+    docker:
+      - image: cimg/node:16.13
+    environment:
+      TRACING_SERVICE_NAME: ci-lint
+      TRACING_CONSOLE_EXPORTER_ENABLED: true
+    steps:
+      - base-install
+      - run:
+          name: Linting
+          command: |
+            PACKAGES=(\
+            'fxa-shared' \
+            'fxa-auth-server' \
+            )
+            for p in "${PACKAGES[@]}"; do
+              (cd packages/$p && yarn lint)
+            done
```

Comment:
Should we use something like this to see if we can do the CI linting in parallel?
```sh
yarn workspaces foreach --include=fxa-auth-server --include=fxa-shared --parallel run lint
# OR
yarn workspaces foreach --include="{fxa-auth-server,fxa-shared}" --parallel run lint
```

Reply:
Since this destroys any formatting/coloring and makes it harder to parse the output here, I'd like to hold off on this. At the moment lint takes only 18 seconds of job time.

Would be happy to re-address this later if we find that this starts to grow to more than 2/3 minutes.